modular_skyrat/modules/modular_items/lewd_items/code/verbs.dm

```diff
@@ -16,3 +16,18 @@
 	. = ..()
 	if(CONFIG_GET(flag/disable_erp_preferences))
 		verbs -= /mob/living/carbon/human/verb/climax_verb
+
+/mob/living/carbon/human/verb/safeword()
+	set name = "Remove Lewd Items"
+	set category = "OOC"
+	set desc = "Removes any and all lewd items from you."
+
+	log_message("[key_name(src)] used the Remove Lewd Items verb.")
```

Comment:
I'm not certain if it does it by default, that'd have to be checked, but it could probably be useful to also add the coordinates and area macro to this, so the logging is even more useful.

Reply:
Sure thing, let me see what I can do.

Reply:
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/68373373/e661fa55-5780-487a-a29f-0217f710d20e)